pkg/internal/cryptosetup/cryptosetup.go

```diff
@@ -112,7 +112,7 @@ func prepareKeyHandle(storeProvider storage.Provider, keyManager kms.KeyManager,
 	var kh *keyset.Handle
 
 	keyIDBytes, err := keyIDStore.Get(keyIDDBKeyName)
-	if err != nil {
+	if err != nil { //nolint:nestif
```

Comment:
Is there any plan to fix minor lint issues like this in next PRs ?

Reply:
I can create an issue and refactor these minor ones